core/domain/skill_services.py

```diff
@@ -460,8 +454,7 @@ def apply_change_list(skill_id, change_list, committer_id):
                 # latest schema version. As a result, simply resaving the
                 # skill is sufficient to apply the schema migration.
                 continue
-            else:
```

Comment:
Ditto, why were these removed?

Reply:
The change_list will be a list(SkillChange). And the case below for invalid change dict is raised from the SkillChange class. See here:
https://github.com/oppia/oppia/blob/9b5d43d18d8644a9f652788d0b4e07f8d3287fe5/core/domain/skill_domain.py#L142

Reply:
Ah, ok, fine then. :)